compiler/rustc_codegen_cranelift/Cargo.lock

```diff
@@ -16,15 +16,15 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.86"
+version = "1.0.94"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "b3d1d046238990b9cf5bcde22a3fb3584ee5cf65fb2765f454ed428c7a0063da"
+checksum = "c1fd03a028ef38ba2276dce7e33fcd6369c158a1bca17946c4b1b701891c1ff7"
```

Comment:
The lockfile changes should be reverted.

Reply:
Yeah, the entire second commit is just for testing and will be dropped. My plan is once you approve the Cranelift changes and somebody from GCC has looked that portion over, I will merge https://github.com/rust-lang/compiler-builtins/pull/735, drop the second commit (which add my crates-io` patch), replace it with a normal `compiler-builtins` bump, and then this PR's merge will change everything atomically.

However, I am stuck completing the current bump at https://github.com/rust-lang/rust/pull/135180, so I will hold off until that gets fixed.